### PR TITLE
Feature/use new yk utils package

### PR DESCRIPTION
**Pull request by vitormcpedro**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
         "six",
         "requests",
         "setuptools"
```
Comment (jppdpf) [inline setup.py:23]: There is a missing comma between "setuptools" and "yk-utils".

File: yk_bit/models/utils.py [hunk 2]
```diff
@@ -23,10 +23,10 @@ def _deserialize(data, klass):
         return deserialize_date(data)
     elif klass == datetime.datetime:
         return deserialize_datetime(data)
-    elif type(klass) == typing.GenericMeta:
-        if klass.__extra__ == list:
+    elif hasattr(klass, '__origin__'):
+        if klass.__origin__ == list:
```
Comment (jppdpf) [inline yk_bit/models/utils.py:27]: I ran this in Python3.6 and I got troubles. I believe we should add this second condition in this if statement "or klass.__origin__ == typing.List". This resolved the issue in 3.6. Also ran this fix in Python3.10 and didn't get a problem.
https://github.com/spec-first/connexion/issues/739#issuecomment-514198578

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,17 +5,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [unreleased]
+## v1.1.0
```
Comment (jppdpf) [inline CHANGELOG.md:8]: We should add the date of the new release.
Isn't supposed for the tag "unreleased" be kept?

File: yk_bit/models/utils.py [hunk 4]
```diff
@@ -23,10 +23,10 @@ def _deserialize(data, klass):
...
             return _deserialize_list(data, klass.__args__[0])
-        if klass.__extra__ == dict:
+        if klass.__origin__ == dict:
             return _deserialize_dict(data, klass.__args__[1])
     else:
         return deserialize_model(data, klass)
```
[the remark this answers is not in the document]
Comment (vitormcpedro) [reply yk_bit/models/utils.py:32]: Yes, I can move it :)

File: CHANGELOG.md (hunk 3 above)
Comment (jppdpf) [inline CHANGELOG.md:8]: I think it should be "[1.1.0]" instead of "v1.1.0". Do you agree?

**Review by jppdpf — APPROVED** [review]

**Pull request merged**